### PR TITLE
Add Tag component and stories

### PR DESCRIPTION
**Pull request by ashleyvansp**
## Description of the change

- Add Tag component for visual dev
- Add stories for long and short tags

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [X ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: (PUT IT HERE)
 - Process.st launch checklist: (PUT IT HERE)

### Development

- [X ] [Prettier](https://www.npmjs.com/package/prettier) was run
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist
- [ ] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/visual-dev/src/components/Tag/Styles.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  padding: 3px 7px;
+  font-weight: 400;
+  font-size: 14px;
+  font-style: normal;
+  line-height: 20px;
+  font-family: "Helvetica Neue", Helvetica, sans-serif;
+  color: #232323;
+`;
+
+export const iconSegment = css`
```
Comment (noahwc) [inline packages/visual-dev/src/components/Tag/Styles.ts:26]: One small update, change the cursor to a pointer when hovering over the button section of the tag to indicate a click action.

**Review by noahwc — CHANGES_REQUESTED** [review]

File: packages/visual-dev/src/components/Tag/Styles.ts (hunk 1 above)
Comment (Locrian24) [reply packages/visual-dev/src/components/Tag/Styles.ts:26]: Haha we talked about this during pair-programming. Technically the pointer cursor is supposed to only be used for hyperlinks (basically to indicate you're leaving the page) and correct hover states should be what indicates a click action/interaction, but this rule is broken a lot in other sites so I guess we should decide where to draw the line. I'm kind of a purist with cursors but that's just me.

Comment (Locrian24) [reply packages/visual-dev/src/components/Tag/Styles.ts:26]: I do think there should be a hover state for the icon segment though (darker background or w/e)

**Review by Locrian24 — CHANGES_REQUESTED** [review]
Hover state on icon segment pls 🙏🏻

File: packages/visual-dev/src/components/Tag/Styles.ts (hunk 1 above)
Comment (ashleyvansp) [reply packages/visual-dev/src/components/Tag/Styles.ts:26]: Good call, I made changes for hover state :+1:

**Review by Locrian24 — APPROVED** [review]
👍🏻

**Review by noahwc — APPROVED** [review]

**Pull request merged**